apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -44,7 +44,10 @@ export default class ModerationNotificationList extends Component {
     const userFetch = await fetch(userUrl)
     const userNotifications = await userFetch.json()
 
-    this.setState({ notifications: [...aiNotifications, ...userNotifications], isLoaded: true })
+    this.setState(
+      { notifications: [...aiNotifications, ...userNotifications] },
+      () => this.setState({ isLoaded: true })
```

Comment:
this might look weird at first sight, i just read in the docs that you can add a callback as a second argument, which waits for the first argument (setting the notifications) to be finished. does that make sense?

Reply:
It's still happening (but only every 20-30th time or so I switch). Also, it doesn't change after a few seconds.
![image](https://user-images.githubusercontent.com/17978375/140949060-7526f134-a902-4f4c-9d9d-607c5e4c570a.png)

 If I reload, it switches the filter to "All" and shows all notifications.

Reply:
@sabinammm reload behaviour is expected else we would have to set cookies